catalog/ui/search-ui/standard/src/test/java/org/codice/ddf/ui/searchui/standard/properties/ConfigurationStoreTest.java

```diff
@@ -33,11 +31,12 @@
 
 public class ConfigurationStoreTest {
 
+  private static final String SERVLET_PATH = "/proxy";
+
   private static final String PROXY_SERVER = "http://www.example.com/wms";
 
-  private static final List<String> IMAGERY_PROVIDERS =
-      Arrays.asList(
-          "{\"type\" \"OSM\" \"url\" \"http://otile1.mqcdn.com/tiles/1.0.0/map\" \"fileExtension\" \"jpg\" \"alpha\" 1},{\"type\" \"OSM\" \"url\" \"http://otile1.mqcdn.com/tiles/1.0.0/sat\" \"fileExtension\" \"jpg\" \"alpha\" 0.5}");
+  private static final String IMAGERY_PROVIDERS =
+      "{\"type\" \"WMS\" \"url\" \"http://example.com\" \"layers\" [\"layer1\" \"layer2\"] \"parameters\" {\"FORMAT\" \"image/png\" \"VERSION\" \"1.1.1\"} \"alpha\" 0.5}";
```

Comment:
This test imageryProvider configuration was updated to match the example in the metatype description.